install_config/upgrading/migrating_embedded_etcd.adoc

```diff
@@ -0,0 +1,72 @@
+[[install-config-upgrading-etcd-data-migration]]
+= Migrating embedded etcd to external etcd
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+Until {product-title} 3.6 it was possible to deploy a cluster with an embedded etcd.
+Since {product-title} 3.7 it is no longer possible.
+Additionaly, the etcd API version since {product-title} 3.6 defaults to v3
+and since {product-title} 3.7 the v3 is the only version allowed.
+Thus, older deployments with embedded etcd with the etcd API version v2 need to migrate
+to the external etcd first
+xref:../../install_config/upgrading/migrating_etcd.adoc[followed by data migration]
+before they can be upgrade to {product-title} 3.7.
+
+This migration process performs the following steps:
+
+- Stop the master service
+- Perform an etcd backup of embedded etcd
+- Deploy external etcd (on the master or new host)
+- Perform backup of original etcd master certificates
+- Generate new etcd certificates for the master
+- Transfer embedded etcd backup to the external etcd host
+- Start external etcd from the transfered etcd backup
+- Re-configure master to use the external etcd
+- Start master
+
+[[etcd-embedded-migration-automated]]
+== Running the Automated Migration Playbook
+
+Either migration to external rpm etcd or external containerized etcd is currently supported.
```

Comment:
Lets add a statement that says in order to perform the migration on atomic host you must be running atomic host 7.4 or later as a solution to https://bugzilla.redhat.com/show_bug.cgi?id=1509195